solana/programs/example-native-token-transfers/src/instructions/release_inbound.rs

```diff
@@ -78,38 +89,65 @@ pub fn release_inbound_mint(
     }
 
     assert!(inbox_item.release_status == ReleaseStatus::Released);
```

Comment:
@kcsongor I think you can delete this `assert` as well given the behaviour of `try_release`. It looks a bit funny to have it in one path/mode but not the other. I think we should do both or neither.

It can be done in a separate cleanup PR though. We don't need to derail this PR.

Reply:
yeah good catch. I think maybe doing it in both cases is fine, even though it's a redundant check (that's why it's an assert with no custom error, as it should never be hit)